src/settings/settings_callbacks.cpp

```diff
@@ -76,7 +76,7 @@ void Callbacks::MetricsTransaction(void *const old_value, void *const new_value,
   action_context->SetState(common::ActionState::IN_PROGRESS);
   bool new_status = *static_cast<bool *>(new_value);
   if (new_status)
-    db_main->GetMetricsManager()->EnableMetric(metrics::MetricsComponent::TRANSACTION, 0);
+    db_main->GetMetricsManager()->EnableMetric(metrics::MetricsComponent::TRANSACTION, 100);
```

Comment:
Just curious why this is set to 100 but the other are at 0? (maybe worth a comment)

Reply:
Originally the consideration was that the time for transaction begin/commit is much faster than the tracker itself, so we apply sampling for them to reduce overhead. But since we don't execute that many transactions per second anyway, I changed this back to 0 for now.